fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/plans/RecordQueryCoveringIndexPlan.java

```diff
@@ -215,8 +215,9 @@ public int getComplexity() {
     }
 
     @Override
-    public int planHash() {
-        return indexPlan.planHash();
+    public int planHash(PlanHashKind hashKind) {
+        // TODO: Is this right?
```

Comment:
this is not correct. we should modify the planHash (for both) to incorporate something indicating that this is covering. this is a bug.